index.js

```diff
@@ -602,6 +635,81 @@ function packAddress(derivationPath, xpub, hdPassphrase, derivationScheme) {
   )
 }
 
+function getAddressHeader(addressType, networkId) {
```

Comment:
protocolMagic and networkId are the same thing - I would unify it and since I don't see no strong reason to change the original terminology, I would suggest to keep it as `protocolMagic`

Reply:
ok, after a second thought, let's keep networkId instead so we are consistent with Trezor which should in turn be consistent with latest Cardano terminology